jetty-http2/http2-hpack/src/main/java/org/eclipse/jetty/http2/hpack/HpackEncoder.java

```diff
@@ -75,18 +74,15 @@
             HttpHeader.AUTHORIZATION,
             HttpHeader.SET_COOKIE,
             HttpHeader.SET_COOKIE2);
-    private static final PreEncodedHttpField CONNECTION_TE = new PreEncodedHttpField(HttpHeader.CONNECTION, "te");
-    private static final PreEncodedHttpField TE_TRAILERS = new PreEncodedHttpField(HttpHeader.TE, "trailers");
-    private static final Trie<Boolean> specialHopHeaders = new ArrayTrie<>(6);
+    private static final EnumSet<HttpHeader> HEADERS_TO_REMOVE = EnumSet.of(HttpHeader.CONNECTION, HttpHeader.KEEP_ALIVE,
```

Comment:
IGNORED_HEADERS would be a better name.
Plus you need to keep naming uniform, either have the __ prefix or remove it for the other statics